data/tests.yml

```diff
@@ -846,9 +846,10 @@
   skip_kotlin: true
   tags:
     highway: "road"
-    lanes: "3"
     sidewalk: "no"
     shoulder: "no"
+    lanes: "3"
+    lanes:forward: "2"
```

Comment:
can this not be inferred from bus:lanes:forward
what happens when this is removed?

Reply:
It could, but `lanes:forward` should be tagged in OSM anyway. Inferring it is being a bit fancy and I'm not trying to solve the entirety of lane tagging in this PR.

Reply:
> but lanes:forward should be tagged in OSM anyway

it _should_ but it often isn't, we shouldn't depend on perfect OSM tagging.
we can leave it in to simplify the implementation for now, but please add a comment describing why this data is redundant so that we can try to remove it later?